project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -1392,6 +1399,72 @@ function c4m_og_get_user_owned_groups($uid, $node_type) {
 }
 
 /**
+ * Add the `Authoring information` tab to the group content editing forms.
+ *
+ * Allows group owner and administrators to change the author of content in the
+ * group. Limits possible authors to group members or anonymous user.
+ */
+function c4m_og_add_content_author_options_to_power_users(&$form) {
+  // If the user is a site administrator, there is no need to do anything.
+  $site_admin_role = user_role_load_by_name('administrator');
+  if (user_has_role($site_admin_role->rid)) {
+    return;
+  }
+
+  list(, $gid) = array_values(og_context());
```

Comment:
please use the standard way to get the group id.
```php
if (!$group = og_context()) {
  return;
}
```

then just use `$group['gid']`